sim/priest/priest.go

```diff
@@ -206,3 +201,90 @@ func New(char *core.Character, selfBuffs SelfBuffs, talents string) *Priest {
 type PriestAgent interface {
 	GetPriest() *Priest
 }
+
+// Priest Specific Aura MOD Handling
+// Other flags
+type PriestSpellFlag uint64
+
+// Returns whether there is any overlap between the given masks.
+func (se PriestSpellFlag) Matches(other PriestSpellFlag) bool {
+	return (se & other) != 0
+}
+
+const (
+	PriestSpellFlagNone  PriestSpellFlag = 0
```

Comment:
this should probably just be int64 if you're constantly casting it to that anyway